changes/2433.feature.rst

```diff
@@ -0,0 +1 @@
+It's now easier to change the action for a Toga command to point to a different handler without having to jump through hoops to create the correct ActionHandler.
```

Comment:
"jump through hoops" is a little pejorative (and an idiom that might not be clear to some readers):
```suggestion
An action for a Toga command can now be easily modified after initial construction.
```